geopandas_view/view.py

```diff
@@ -461,8 +463,24 @@ def _choropleth(
     **kwds : dict
         Keyword arguments to pass to folium.GeoJson
     """
+    from branca.utilities import color_brewer
+
+    nan_fill_color = kwds.get('nan_fill_color') or "black"
```

Comment:
```suggestion
    nan_fill_color = kwds.pop('nan_fill_color', "black")
```

This does the same thing and removes the key from the dict (so we are not passing it to GeoJson kwds). Same below.